apps/studio/components/interfaces/ProjectCreation/ComputeSizeSelector.tsx

```diff
@@ -0,0 +1,121 @@
+              <>
+                <p>
+                  The size for your dedicated database. You can change this later. Learn more about{' '}
+                  <InlineLink
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    className="text-inherit hover:text-foreground transition-colors"
+                    href={`${DOCS_URL}/guides/platform/compute-add-ons`}
+                  >
+                    compute add-ons
+                  </InlineLink>{' '}
+                  and{' '}
+                  <InlineLink
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    className="text-inherit hover:text-foreground transition-colors"
+                    href={`${DOCS_URL}/guides/platform/manage-your-usage/compute`}
+                  >
+                    compute billing
+                  </InlineLink>
+                  .
+                </p>
+              </>
+            }
+          >
+            <Select_Shadcn_ value={field.value} onValueChange={(value) => field.onChange(value)}>
+              <SelectTrigger_Shadcn_ className="[&_.instance-details]:hidden">
+                <SelectValue_Shadcn_ placeholder="Select a compute size" />
+              </SelectTrigger_Shadcn_>
+              <SelectContent_Shadcn_>
+                <SelectGroup_Shadcn_>
+                  {sizes
+                    .filter((option) =>
+                      instanceSizeSpecs[option].cloud_providers.includes(
+                        form.getValues('cloudProvider') as CloudProvider
+                      )
+                    )
+                    .map((option) => {
+                      return (
+                        <SelectItem_Shadcn_ key={option} value={option}>
+                          <div className="flex flex-row i gap-2">
+                            <div className="text-center w-[80px]">
+                              <Badge
+                                variant={option === 'micro' ? 'default' : 'brand'}
+                                className="rounded-md w-16 text-center flex justify-center font-mono uppercase"
+                              >
+                                {instanceSizeSpecs[option].label}
+                              </Badge>
+                            </div>
+                            <div className="text-sm">
+                              <span className="text-foreground">
+                                {instanceSizeSpecs[option].ram} RAM /{' '}
+                                {instanceSizeSpecs[option].cpu}{' '}
+                                {getCloudProviderArchitecture(
+                                  form.getValues('cloudProvider') as CloudProvider
+                                )}{' '}
+                                CPU
+                              </span>
+                              <p
+                                className="text-xs text-foreground-light instance-details"
```

Comment:
If `instance-details` is used only for element selection and not for styling, can we put it in a data attribute?